<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Initial_adjustment</class>
 <widget class="QDialog" name="Initial_adjustment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Initial Adjustment</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="9">
    <widget class="QLabel" name="label_image">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>No Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>368</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: none;
	padding: 0px;
	margin: 0px;
}

QGroupBox::title {
	color: #000036;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="b_close">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #3e37ff;
    border: none;
    border-radius: 10px;
    padding: 10px; /* Upraví vnitřní prostor tlačítka */
	color: white;
}

QPushButton:hover {
    background-color: #000036;
}</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="b_setup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>5000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #3e37ff;
    border: none;
    border-radius: 10px;
    padding: 10px; /* Upraví vnitřní prostor tlačítka */
	color: white;
}

QPushButton:hover {
    background-color: #000036;
}

QPushButton:disabled {
    background-color: #808080; /* Šedá barva pro deaktivované tlačítko */
    color: #c0c0c0; /* Světlejší šedá barva textu */
    border: none;
}</string>
        </property>
        <property name="text">
         <string>Set up</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>368</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: none;
	padding: 0px;
	margin: 0px;
}

QGroupBox::title {
	color: #000036;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="b_freedrive_on">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #3e37ff;
    border: none;
    border-radius: 10px;
    padding: 10px; /* Upraví vnitřní prostor tlačítka */
	color: white;
}

QPushButton:hover {
    background-color: #000036;
}

QPushButton:disabled {
    background-color: #808080; /* Šedá barva pro deaktivované tlačítko */
    color: #c0c0c0; /* Světlejší šedá barva textu */
    border: none;
}</string>
           </property>
           <property name="text">
            <string>Freedrive ON</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="b_freedrive_off">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #3e37ff;
    border: none;
    border-radius: 10px;
    padding: 10px; /* Upraví vnitřní prostor tlačítka */
	color: white;
}

QPushButton:hover {
    background-color: #000036;
}

QPushButton:disabled {
    background-color: #808080; /* Šedá barva pro deaktivované tlačítko */
    color: #c0c0c0; /* Světlejší šedá barva textu */
    border: none;
}</string>
           </property>
           <property name="text">
            <string>Freedrive OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="b_light_on_init">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #3e37ff;
    border: none;
    border-radius: 10px;
    padding: 10px; /* Upraví vnitřní prostor tlačítka */
	color: white;
}

QPushButton:hover {
    background-color: #000036;
}

QPushButton:disabled {
    background-color: #808080; /* Šedá barva pro deaktivované tlačítko */
    color: #c0c0c0; /* Světlejší šedá barva textu */
    border: none;
}</string>
           </property>
           <property name="text">
            <string>Light ON</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="b_light_off_init">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #3e37ff;
    border: none;
    border-radius: 10px;
    padding: 10px; /* Upraví vnitřní prostor tlačítka */
	color: white;
}

QPushButton:hover {
    background-color: #000036;
}

QPushButton:disabled {
    background-color: #808080; /* Šedá barva pro deaktivované tlačítko */
    color: #c0c0c0; /* Světlejší šedá barva textu */
    border: none;
}</string>
           </property>
           <property name="text">
            <string>Light OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#000036;</string>
        </property>
        <property name="text">
         <string>Position the robot and camera so that the camera is perpendicular to the calibration pad. The calibration pad must be in focus and aligned in a rectangle. The size of the rectangle can be partially changed using the slider below.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
	border: none;
	padding: 0px;
	margin: 0px;
}

QGroupBox::title {
	color: #000036;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_distance">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">margin: 20px;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <fontweight>DemiBold</fontweight>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#000036;</string>
           </property>
           <property name="text">
            <string>Enter the approximate distance from the calibration pad (mm):</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSlider" name="Slider_rectangle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider  {
    margin: 20px 50px 20px 50px;
}

QSlider::groove:horizontal {
    border: 1px solid #999;
    height: 20px;
    background: #ccc;
    border-radius: 10px;
}

QSlider::handle:horizontal {
    background: #0078D7;
    border: 1px solid #005A9E;
    width: 18px;
    margin: 0px;
    border-radius: 9px;
}</string>
        </property>
        <property name="minimum">
         <number>30</number>
        </property>
        <property name="maximum">
         <number>70</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Slider_rectangle</tabstop>
  <tabstop>lineEdit_distance</tabstop>
  <tabstop>b_freedrive_on</tabstop>
  <tabstop>b_freedrive_off</tabstop>
  <tabstop>b_light_on_init</tabstop>
  <tabstop>b_light_off_init</tabstop>
  <tabstop>b_setup</tabstop>
  <tabstop>b_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
